hw/pci-bridge/simbridge.c

```diff
@@ -32,6 +32,10 @@
 #define XIO3130_EXP_OFFSET              0x90
 #define XIO3130_AER_OFFSET              0x100
 
+#define ADDR_ALIGN_BYTES	4
+#define WARN_UNALIGNED(addr) \
+        QEMU_IS_ALIGNED(addr, ADDR_ALIGN_BYTES) ? "" : "(WARN unaligned address)"
```

Comment:
Agree with your suggestion to remove WARN and make this "(unaligned)".  Useful debug but not too alarming.